src/main/java/ro/msg/learning/shop/testprofileConfigureSeparateDataSource/OrderJpaConfig.java

```diff
@@ -0,0 +1,32 @@
+package ro.msg.learning.shop.testprofileConfigureSeparateDataSource;
```

Comment:
please follow the naming conventions for packages

Reply:
Done - using lower case and "_"